content/gr/docs/chart_best_practices/_index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Βέλτιστες Πρακτικές"
+sidebar: true
+weight: 4
+aliases: ["/docs/topics/chart_best_practices/"]
```

Comment:
The aliases are only used to redirect pages that have moved. In this case, this would accidentally redirect people looking for an English doc to this Greek page, so we definitely need to remove this line.

Reply:
Thanks for pointing that out @bridgetkromhout ! I fixed it.